### ISSUE TITLE
2FA QR Code - Almost impossible to read

### ISSUE THREAD

**Issue by wolfgangvc** [StylishThemes/GitHub-Dark#557]
Setting up 2 factor authentication is almost impossible while using GitHub-Dark

* **Browser**: Firefox
* **Operating System**: Ubuntu 16.04
* **Screenshot**: Obscured due to security issues
![image](https://user-images.githubusercontent.com/20362791/32944202-9a2a8412-cb86-11e7-9b58-65260b6f5c8e.png)

The dark theme results in the qr code for 2FA being essentially black on black.

**Suggested fix:**

``` CSS
.qr-code-table{
  background-color: #FFF!important;
}  
.qr-code-table .white{
  background-color: #FFF!important;
}
```
Or similar.

**Comment by silverwind** [comment]
Can you paste the HTML of that box? (Right-Click, Inspect, Copy (Outer) HTML in the inspector)

**Comment by wolfgangvc** [comment]
I have cut the html short (as its VERY VERY long, but very repetetive)

It is using a table to build up the qrcode grid.

```HTML
<div class="two-factor-qr-code flex-justify-center">
    <table class="qr-code-table" cellspacing="0" cellpadding="0">
        <tbody>
            <tr>
                <td class="black"></td>
                <td class="black"></td>
                <td class="black"></td>
                <td class="black"></td>
                <td class="black"></td>
                <td class="black"></td>
                <td class="black"></td>
                <td class="white"></td>
                <td class="black"></td>
                <td class="black"></td>
                ..... (etc)
            </tr>
            <tr>
                <td class="black"></td>
                <td class="white"></td>
                <td class="white"></td>
                <td class="white"></td>
                <td class="white"></td>
                <td class="white"></td>
                <td class="black"></td>
                <td class="white"></td>
                <td class="black"></td>
                <td class="black"></td>
                ..... (etc)
            </tr>
            ..... (etc)
        </tbody>
    </table>
</div>
```

**Comment by silverwind** [comment]
Could you also include the outer box with the white border? Length does not matter, you can paste it on https://gist.github.com/.

**Comment by silverwind** [comment]
Nevermind, I was able to find that page:

https://github.com/settings/two_factor_authentication/verify

**Comment by silverwind** [comment]
And fixed in 1.20.4. I guess a QR code is the only place where we don't want to mess with colors 😆
